.github/workflows/main.yml

```diff
@@ -12,6 +12,17 @@ jobs:
         run: bash .ci/build.sh build
         env:
           BENCHMARK_HOME: env.GITHUB_WORKSPACE
+  Unit-Tests-MacOS:
```

Comment:
Since this is identical to the above, can we use a matrix? Example: https://github.com/opensearch-project/opensearch-build/blob/main/.github/workflows/python-tests.yml#L8

Reply:
Thanks for pointing it out, I corrected the config